libp2p/utils.py

```diff
@@ -77,7 +77,7 @@ def encode_delim(msg: bytes) -> bytes:
         raise ParseError(f"`len(msg_bytes)` should not be 0")
     if msg_bytes[-1:] != b"\n":
         raise ParseError(
-            f'`msg_bytes` is not delimited by b"\\n": `msg_bytes`={msg_bytes}'
+            f'`msg_bytes` is not delimited by b"\\n": `msg_bytes`={msg_bytes!r}'
```

Comment:
This one can also be added to that separate PR.

Reply:
Also fixed in d52b093286c9b4c33b26bb9d4af9ee5a24e52c71 on `master` already